CIP-0055/README.md

```diff
@@ -1,12 +1,16 @@
 ---
 CIP: 55
 Title: Protocol Parameters (Babbage Era)
-Authors: Jared Corduan <jared.corduan@iohk.io>
 Status: Active
-Type: Informational
+Category: Ledger
+Authors:
+  - Jared Corduan <jared.corduan@iohk.io>
+Implementors:
+  - IOG
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/265
 Created: 2022-05-19
 License: CC-BY-4.0
```

Comment:
mismatch with the Copyright section 🤓

Reply:
also failing to follow Apache precedent set by earlier protocol parameter CIPs and continued in https://github.com/cardano-foundation/CIPs/pull/650.  Have fixed it in https://github.com/cardano-foundation/CIPs/pull/707/commits/784e8775ee8cd40deb041d8f84d3df6ed3f22213.

Reply:
great work, as always Robert :)